src/Components/Border.js

```diff
@@ -0,0 +1,12 @@
+const Border = () => {
+    return (
+        <div className="border-container">
+        <div className="border top"></div>
+        <div className="border right"></div>
+        <div className="border bottom"></div>
+        <div className="border left"></div>
+      </div>
+    );
+};
+
```

Comment:
I want to know how this works?? I love the idea of reusing styled elements in this way, but can't wrap my head around how it functions.

Reply:
this is just a rectangle div on each corner of the browser, and because I have it on my return in app.js it shows up on all pages!